string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,31 @@
+/* exported numVowels */
+// numVowels
+// input: String
+// output: Number of vowels in passed String
+
+// PSEUDOCODE
+// create storage for output
+// traverse every character in String
+//  -change character to lowercase and check if vowel
+//    -if so, increment output
+//      -break
+//    -if not, break
+// return output
+
+function numVowels(string) {
+  var output = 0;
+  for (var i = 0; i < string.length; i++) {
+    switch (string[i].toLowerCase()) {
+      case 'a':
+      case 'e':
+      case 'i':
+      case 'o':
+      case 'u':
+        output++;
+        break;
+      default:
+        break;
```

Comment:
You should be able to omit this. If the default is to do nothing, then no `default` case is needed, since that's what would happen anyway.